app/src/main/res/values/strings.xml

```diff
@@ -449,4 +446,9 @@
   <string name="correct_submitted_answer_with_append">Correct submitted answer: %s</string>
   <string name="incorrect_submitted_answer">Incorrect submitted answer</string>
   <string name="incorrect_submitted_answer_with_append">Incorrect submitted answer: %s</string>
+
+<!--  Help Activity-->
```

Comment:
Nit
1. There should be space at the start.
2. And there should be 1 space before and after text.

`  <!-- Help Activity -->`

Reply:
Thanks sir, I've done some changes in file, please review it again

Reply:
@TheSwarnim It does not match with the above comment. Can you please take a look.

Reply:
Also, update your branch by merging with latest develop.